src/Symfony/Component/HttpFoundation/ResponseHeaderBag.php

```diff
@@ -221,7 +229,7 @@ public function getCookies(string $format = self::COOKIES_FLAT): array
      */
     public function clearCookie(string $name, ?string $path = '/', ?string $domain = null, bool $secure = false, bool $httpOnly = true, ?string $sameSite = null /* , bool $partitioned = false */): void
     {
-        $partitioned = 6 < \func_num_args() ? \func_get_arg(6) : false;
+        $partitioned = 6 < \func_num_args() ? func_get_arg(6) : false;
```

Comment:
not related, cs fixer insists on this change even though i did not change this line...

(running cs fixer over the whole project, there is a bunch of changes - might be good to do that globally to avoid random changes in every pull request)